Makefile

```diff
@@ -133,8 +136,14 @@ uninstall: manifests kustomize ## Uninstall CRDs from the K8s cluster specified
 	$(KUSTOMIZE) build config/crd | kubectl delete -f -
 
 deploy: manifests kustomize ## Deploy controller to the K8s cluster specified in ~/.kube/config.
+	sed -i "" "s/namespace: splunk-operator/namespace: ${NAMESPACE}/g"  config/default/kustomization.yaml
+	sed -i "" "s/value: WATCH_NAMESPACE_VALUE/value: \"${WATCH_NAMESPACE}\"/g"  config/default/kustomization.yaml
+	sed -i "" "s|SPLUNK_ENTERPRISE_IMAGE|${SPLUNK_ENTERPRISE_IMAGE}|g"  config/default/kustomization.yaml
```

Comment:
Just for my understanding, what is this particular change for?

Reply:
Idea is to take splunk enterprise image version on command line as argument and then pass it while generating the yaml file. 
`make generate-artifacts IMG=splunk/splunk-operator:1.1.0 SPLUNK_ENTERPRISE_IMAGE=splunk/splunk:8.2.6 WATCH_NAMESPACE="" VERSION=1.1.0 NAMESPACE=splunk-operator`